### ISSUE TITLE
ean-128/gs1-128 support?

### ISSUE THREAD

**Issue by adsad213** [WhyNotHugo/python-barcode#36]
since gs1-128 is based on code128, could this possibly support it?

**Comment by MDziwny** [comment]
I would be very interested too.

**Comment by wellmonwu** [comment]
it is better to support ean-128 and  print to doc/docx.

**Comment by MDziwny** [comment]
Actually it can be generated with the `Code128` generator by prepending the FNC1 character `\xf1` to the code generated.

For instance:
``` python
import barcode

GS1_FNC1_CHAR = u'\xf1'

code = u"00123456780000000001"

code_128_creator = barcode.get_barcode_class("code128")
gs1_128_code = code_128_creator(GS1_FNC1_CHAR + code)
```

**Comment by MDziwny** [comment]
I could make a PR to make it more user friendly

**Comment by wellmonwu** [comment]
> I could make a PR to make it more user friendly

it would help a lot

**Comment by WhyNotHugo** [comment]
@MDziwny Are you entirely sure there's not another letter _before_ the FNC1 character? (I'm just making sure, since I haven't come across very clear information).

In any case, a PR is more than welcome. A subclass that wraps around the current code128 renderer should work fine, so if you want to address it, I'll be happy to merge it in. Thanks!

**Comment by MDziwny** [comment]
From what I've read, for instance on https://www.gs1-128.info/:

> In essence, **GS1-128 is a derivative language of the Code 128** symbology, utilizing a flagging character (Function Code 1 (FNC1)) and application identifiers.

For the letter before the FNC1, it is already inserted by Code-128 as the first character of the code.

I will do the PR soon.

**Comment by WhyNotHugo** [comment]
Fixed via #37. Version 0.10.0 released with this (and a few other) changes.
